xml/ha_troubleshooting.xml

```diff
@@ -129,8 +129,14 @@
     <term>Where to find the log files? </term>
     <listitem>
      <para>
+      <remark>taroth 2018-03-05: ygao, should I replace the following with
+     "Pacemaker writes its logs to '/var/log/pacemaker/pacemaker.log' and
+     '/var/log/pacemaker/bundles/*/pacemaker.log, respectively"?
+     (whatever the bundle logs are...), see c#4</remark>
```

Comment:
Yes for the main log /var/log/pacemaker/pacemaker.log. Since we haven't yet documented bundle feature itself, probably we can ignore bundle logs in here for now.

Or probably we can just put it like "Pacemaker writes its logs under '/var/log/pacemaker' directory and the main log file is '/var/log/pacemaker/pacemaker.log'.

Reply:
Good idea, I followed your suggestion. :)